pkg/steps/release/create_release.go

```diff
@@ -102,6 +105,61 @@ func (s *assembleReleaseStep) Run(ctx context.Context, dry bool) error {
 		return nil
 	}
 
+	if _, err := s.saGetter.ServiceAccounts(s.jobSpec.Namespace).Create(&coreapi.ServiceAccount{
+		ObjectMeta: meta.ObjectMeta{
+			Name:      "ci-operator",
+			Namespace: s.jobSpec.Namespace,
+		},
+	}); err != nil && !errors.IsAlreadyExists(err) {
+		return fmt.Errorf("could not create service account '%s' for: %v", "ci-operator", err)
```

Comment:
Why bother print-formatting when it's a static string?

Reply:
Done.